### PR TITLE
display usernames

### PR DESCRIPTION
**Pull request by Hou5ton**
#385 
added username call from database instead of id

### CODE REVIEW

File: Network/Public/Include/Network/Primitives.hpp [hunk 1]
```diff
@@ -135,6 +135,8 @@ struct MessageInfo
     std::uint64_t msgID;
     /// sender ID uint64_t variable
     std::uint64_t senderID;
+    /// user Login string variable
+    std::string userLogin;
```
[the remark this answers is not in the document]
Comment (Svidunovich) [reply Network/Public/Include/Network/Primitives.hpp:139]: @StasDash Which one?

**Comment by Svidunovich** [comment]
@Hou5ton To add changes, git pull --rebase origin back/work_with_logins
It pr ready for review and approve.

File: DataAccess/DataAccess.Postgre/PostgreRepository.cpp [hunk 2]
```diff
@@ -34,7 +34,11 @@ std::vector<Network::MessageInfo> PostgreRepository::getMessageHistoryForUser(co
                                    ->join(Utility::SQLJoinType::J_INNER, "channel_msgs", "channel_msgs.msg_id = msgs.msg_id")
                                    ->Where("channel_msgs.channel_id = " + std::to_string(channelID))
                                    ->execute();
-    
+
+    pTable->changeTable("users");
+    auto logins = pTable->Select()->columns({"login"})->execute();
```
Comment (f1nal3) [inline DataAccess/DataAccess.Postgre/PostgreRepository.cpp:39]: Do we need to grab the whole table? Can't we do something like " SELECT login WHERE user-id = sender.id" ?

Comment (f1nal3) [reply DataAccess/DataAccess.Postgre/PostgreRepository.cpp:39]: that would make line 52 much cleaner

**Review by Orange-hanter — APPROVED** [review]

**Comment by Orange-hanter** [comment]
@f1nal3

**Review by f1nal3 — APPROVED** [review]

**Pull request merged**